src/bindings/python/tests/test_graph/test_constant.py

```diff
@@ -205,55 +205,6 @@ def test_init_with_scalar(init_value, src_dtype, dst_dtype, shared_flag, data_ge
     assert np.allclose(const_data, expected_result)
 
 
-@pytest.mark.parametrize(
-    ("src_dtype"),
-    [
-        (np.float16),
-        (np.uint16),
-    ],
-)
-@pytest.mark.parametrize(
-    ("shared_flag"),
-    [
-        (True),
-        (False),
-    ],
-)
-@pytest.mark.parametrize(
-    ("data_getter"),
-    [
-        (DataGetter.COPY),
-        (DataGetter.VIEW),
-    ],
-)
-def test_init_bf16_populate(src_dtype, shared_flag, data_getter):
-    data = np.random.rand(1, 2, 16, 8) + 0.5
-    data = data.astype(src_dtype)
-
-    # To create bf16 constant, allocate memory and populate it:
-    init_data = np.zeros(shape=data.shape, dtype=src_dtype)
-    ov_const = ops.constant(init_data, dtype=Type.bf16, shared_memory=shared_flag)
-    ov_const.data[:] = data
```

Comment:
I think it's good to add a test that checks the error when somebody tries to change data in const node

Reply:
done